### PR TITLE
Updated readme with 1406 1407 1414 chain

### LINKED ISSUE #51: Update readme with 1407, 1406, 1414

### PR DESCRIPTION
**Pull request by gulshanvasnani**
Updated readme with 1406, 1407 and 1414 chain id.

Fixes #51

### CODE REVIEW

**Comment by schemar** [comment]
Reviewing

File: README.md [hunk 1]
```diff
@@ -30,14 +30,16 @@ The default directory for mosaic to store chain data is `~/.mosaic`.
 You can specify a different directory with the `--mosaic-dir` option.
 
 Examples:
-* Starts three containers to follow these chains:
-  * `./mosaic start 200 ropsten rinkeby`
+* Starts five containers to follow these chains:
+  * `./mosaic start ropsten rinkeby 1406 1407 1414`
```
Comment (schemar) [inline README.md]: Hey, sorry this is my fault, but rinkeby cannot be used. Can you please remove rinkeby here?

File: README.md [hunk 2]
```diff
@@ -47,6 +47,13 @@ Examples:
 
 * Lists all running mosaic chain containers:
   * `./mosaic list`
+  
+  ## Available chains
+  ```
+  1406, 1407 - Testnet auxiliary chains.
+  1414 - Production auxiliary chain.
+  ```
```
Comment (schemar) [inline README.md]: please remove the leading spaces as this weirdly indents the paragraph (see in preview; including the header)

Comment (gulshanvasnani) [reply README.md]: @schemar , It was done to make sub-section under `##Mosaic Chains`.

**Pull request merged**